sdk/storage/azure-storage-file-share/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 12.9.0-beta.4 (Unreleased)
+- ShareLeaseClient now updates it's leaseID through a lease change.
```

Comment:
Should we say lease modification? since it applies to change and renew?

Reply:
and acquire

Reply:
I mean, that technically was the behavior already. Acquire and renew didn't technically need the changes but if we ever changed how these clients were initialized they would.

Reply:
ok, I think that makes sense